tasks/install.yml

```diff
@@ -24,6 +24,7 @@
         url: "https://github.com/prometheus/node_exporter/releases/download/v{{ node_exporter_version }}/node_exporter-{{ node_exporter_version }}.linux-{{ go_arch }}.tar.gz"
         dest: "/tmp/node_exporter-{{ node_exporter_version }}.linux-{{ go_arch }}.tar.gz"
         checksum: "sha256:{{ node_exporter_checksum }}"
+        mode: '0666'
```

Comment:
`0666` seems too broad, lets change this to `0644` which

Reply:
makes sense